### DISCUSSION TITLE
Does nextflow have any parameters to set the priority of pods in k8s?

### DISCUSSION THREAD

**Discussion by Zruto** [nextflow-io/nextflow#4079] [Q&A] [answered]
Some tasks are relatively high-priority. Can I set a higher priority when using nextflow to create a pod, or specify to run on certain nodes?

**Comment by mribeirodantas** [comment, marked as answer]
Hello, @Zruto. You should be able to set those things with the [`pod` process directive](https://www.nextflow.io/docs/latest/process.html#pod). Does that work for you?
